regtests/bin/matrix_ncep

```diff
@@ -154,6 +154,8 @@ fi
   export   infgrv='y' # Second harmonic generation tests
   export     uost='y' # ww3_ts4 Unresolved Obstacles Source Term (UOST)
   export    assim='y' # Restart spectra update
+  export     coup='y' # Atmosphere, ocean, and ice coupling
```

Comment:
Are there any secondary library requirements for running these tests?  Also we should specify that this is OASIS coupling, not just "coupling".

Reply:
I'm asking about the requirements, because if there are any, I think this should be a 'n' for NCEP.